connector/connect/server/src/test/scala/org/apache/spark/sql/connect/planner/SparkConnectProtoSuite.scala

```diff
@@ -619,6 +619,50 @@ class SparkConnectProtoSuite extends PlanTest with SparkConnectPlanTest {
     comparePlans(connectPlan1, sparkPlan1)
   }
 
+  test("Test observe") {
```

Comment:
please add negative tests for throwing analysis exceptions when submitting non aggregation functions.

Reply:
This test just test the protocol msg. I can add them into end-to-end place.

Reply:
OK